### DISCUSSION TITLE
Is this project abandoned?

### DISCUSSION THREAD

**Discussion by mezanddav** [glidejs/glide#663] [General]
We are thinking of using Glidejs but when I look at the updates and the piling bug reports I am worried that the project is not maintained. 

It is by far the best framework for sliding features on the internet. Glidejs never failed me and the alternatives are absolute trash.

**Comment by jedrzejchalubek** [comment]
Happy to hear that! Unfortunately, no so much time to actively maintain the project. That said, the nature of it (zero-dependency, etc) will not make Glide stop working or something in the future. If the current set of features is all you need it is still a solid choice today

However, if you looking for something more feature-rich or further developed, I think there may be good options out there

**Reply by mezanddav** [reply]
Is there a possibility where we could work out a way to keep this alive?

License? Donations? 

I was serious about not having good and reliable alternative. They are either reliant on some idiotic library / framework or it's just not up to the standards.

**Reply by jedrzejchalubek** [reply]
I don't think slider can be monetized to the point where it could be a full-time job like other frameworks.

Can you tell me what parts of Glide bother you because of the lack of systematic updates? It would help me much to know what are specific pain points :)

There are no dependencies that need to be updated, use quite modern ES6, etc. It will be still solid code for the next few years. Its modular architecture allows it to basically swap every component of it

**Reply by GuitarKat** [reply]
I apologize for jumping in, but I am wondering about just perhaps examples for certain use cases on when/how to remove based on mobile size/desktop, and other use cases others have brought up - another recent one seems to be different slide sizes (feature request though). There was that one thread that's been going on for three years for glidejs and breakpoints. We have made our own workarounds, which is fine and we're happy for glidejs as it's no dependencies and because of that kept my headache low on any maintenance.

For what this community has been able to do with it, and your guys' code on it, has been really helpful and I haven't liked other sliders.
